packaging/rpm/cri-resource-manager.spec.in

```diff
@@ -35,7 +41,9 @@ install -m 0700 -d %{?buildroot}%{_sharedstatedir}/cri-resmgr
 %{_sysconfdir}/sysconfig/*
 %{_unitdir}/*
 %dir %attr(0700,root,root) %{_sharedstatedir}/cri-resmgr
+%dir %attr(0700,root,root) %{_sysconfdir}/cri-resmgr
+%{_sysconfdir}/cri-resmgr/*.sample
```

Comment:
sample config should go to %doc section of rpm.
here we can put really some default config.

Reply:
Removed `doc` sub-package and added sample config files to the main package, tagged as %doc. The default configuration is installed with `memtier` enabled as the policy.

Added a command-line option (`--reset-policy`) for resetting the policy-specific data from the cache. This can be used to change policy by 1) stopping `cri-resmgr`, 2) updating the config file, 3) resetting the policy, 4) then starting `cri-resmgr` again.

Added another command line option (`--allow-policy-switch`) to allow this to happen automatically if necessary during the startup phase.

Updated the default fallback configuration with instructions about how to change policy manually, or allow it to happen automatically.